isPrime.py

```diff
@@ -0,0 +1,23 @@
+#This is the week 2 assignment for OAF PSD
+#This code checks if a number is prime
+
+def isPrime(number):
+    #this code will not identify if a negative number is prime
+    if number < 0:
+        return False
+    #if the number is one of these, it's prime
+    elif number in [0, 1, 2, 3]:
+        return True
+    #if the number is even, it can't be prime
+    elif (number % 2 == 0):
+        return False
+    #check if anything divides the number
+    else:
+        i = 2
+        while i < number:
+            if number % i == 0:
+                return False
+            else:
+                i += 1
```

Comment:
works, but it's a bit inefficient. No big deal since this is not an algorithms class, but as a hint: you don't need to check one iteration if you already verified that a previous number was not a clean divisor (i.e. 13 is not divisible by 2, so it will never be divisible by 4,6,8,10,...)